extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/Args.scala

```diff
@@ -5,15 +5,15 @@ import caseapp.core.Error.MalformedValue
 import caseapp.core.argparser.{ArgParser, SimpleArgParser}
 import org.broadinstitute.monster.EncodeExtractionBuildInfo
 
-@AppName("ENCODE Extraction Pipeline")
-@AppVersion(EncodeExtractionBuildInfo.version)
-@ProgName("org.broadinstitute.monster.etl.encode.ExtractionPipeline")
 /**
   * Command-line arguments for the ETL workflow.
   *
   * scio's `ContextAndArgs.typed` delegates down to `caseapp`, which will generate
   * parsers + help text for these args (as well as Beams' underlying options)
   */
+@AppName("ENCODE Extraction Pipeline")
+@AppVersion(EncodeExtractionBuildInfo.version)
+@ProgName("org.broadinstitute.monster.etl.encode.ExtractionPipeline")
```

Comment:
Not scalafmt: Putting the docstring under the annotations caused it to be marked as "unmoored", leading to a failure in packaging.

Reply:
good to know